requirements.txt

```diff
@@ -12,13 +12,13 @@
 # scripts/convert-requirements-to-conda-yml.py as though it can only be found
 # on pip.
 
-GDAL>=3.0.4,<=3.1.3
+GDAL==3.1.2
```

Comment:
Does GDAL>3.1.2 break the build?

Reply:
And more generally, can this upper bound be left open (or at least have a version cap set farther out)?

Reply:
Fair question, let me check GDAL's requirements for other versions!

Reply:
Hey @phargogh, it looks like `libgdal 3.1.2` has builds for `PROJ 7.0.0 to 7.1.1` and `libgdal 3.1.3` requires `PROJ 7.1.1+`. I've actually been testing with 7.1.1. Anyway, I think opening the upper bound is fine for now. https://github.com/natcap/invest/pull/350/commits/d823912c0bc045ea7663e67854ef69439548300b